var/spack/repos/builtin/packages/lesstif/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Lesstif(AutotoolsPackage):
+    """LessTif is the Hungry Programmers' version of OSF/Motif."""
+
+    homepage = "https://sourceforge.net/projects/lesstif"
+    url      = "https://sourceforge.net/projects/lesstif/files/lesstif/0.95.2/lesstif-0.95.2.tar.bz2/download"
+
+    version('0.95.2', sha256='eb4aa38858c29a4a3bcf605cfe7d91ca41f4522d78d770f69721e6e3a4ecf7e3')
+
+    depends_on('libice')
+    depends_on('libsm')
+    depends_on('libxt')
+
+    def setup_build_environment(self, env):
+        env.set('DESTDIR', self.prefix)
+
+    def configure_args(self):
+        args = [
+            '--prefix='
+        ]
+        return args
```

Comment:
Can you explain why you needed to set `DESTDIR` and why `--prefix` doesn't work? The [Homebrew package](https://github.com/Homebrew/homebrew-core/blob/master/Formula/lesstif.rb) doesn't do this.

Reply:
The --prefix install doesn't get every file into the PREFIX directory so the install fails, e.g.

`5 errors found in build log:
     1290    Making install in autoconf
     1291    make[2]: Entering directory `/tmp/vagrant/spack-stage/spack-stage-lesstif-0.95.2-v2chyxq4oy4mc4sgt22o2cj47
             rbkf3mk/spack-src/scripts/autoconf'
     1292    make[3]: Entering directory `/tmp/vagrant/spack-stage/spack-stage-lesstif-0.95.2-v2chyxq4oy4mc4sgt22o2cj47
             rbkf3mk/spack-src/scripts/autoconf'
     1293    make[3]: Nothing to be done for `install-exec-am'.
     1294    test -z "/usr/share/aclocal" || mkdir -p -- "/usr/share/aclocal"
     1295     /usr/bin/install -c -m 644 'ac_find_motif.m4' '/usr/share/aclocal/ac_find_motif.m4'
  >> 1296    /usr/bin/install: cannot create regular file '/usr/share/aclocal/ac_find_motif.m4': Permission denied
  >> 1297    make[3]: *** [install-aclocalDATA] Error 1
     1298    make[3]: Leaving directory `/tmp/vagrant/spack-stage/spack-stage-lesstif-0.95.2-v2chyxq4oy4mc4sgt22o2cj47r
             bkf3mk/spack-src/scripts/autoconf'
  >> 1299    make[2]: *** [install-am] Error 2
     1300    make[2]: Leaving directory `/tmp/vagrant/spack-stage/spack-stage-lesstif-0.95.2-v2chyxq4oy4mc4sgt22o2cj47r
             bkf3mk/spack-src/scripts/autoconf'
  >> 1301    make[1]: *** [install-recursive] Error 1
     1302    make[1]: Leaving directory `/tmp/vagrant/spack-stage/spack-stage-lesstif-0.95.2-v2chyxq4oy4mc4sgt22o2cj47r
             bkf3mk/spack-src/scripts'
  >> 1303    make: *** [install-recursive] Error 1

See build log for details:
  /tmp/vagrant/spack-stage/spack-stage-lesstif-0.95.2-v2chyxq4oy4mc4sgt22o2cj47rbkf3mk/spack-build-out.txt
`

Reply:
Looks like I should incorporate some of the Homebrew build recipe; let me push more changes to this today before merging ...

Reply:
Yeah, see if installing in serial helps.